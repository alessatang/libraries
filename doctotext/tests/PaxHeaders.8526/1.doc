30 atime=1389151461.414934548
30 ctime=1389151461.414934548
