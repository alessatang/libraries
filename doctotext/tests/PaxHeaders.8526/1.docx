30 atime=1389151461.415934544
30 ctime=1389151461.415934544
