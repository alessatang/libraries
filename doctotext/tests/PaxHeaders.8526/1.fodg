30 mtime=1384978086.650497407
30 atime=1389151461.424934508
30 ctime=1389151461.424934508
