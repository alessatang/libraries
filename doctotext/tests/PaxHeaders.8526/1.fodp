30 mtime=1384978086.678497417
30 atime=1389151461.648933626
30 ctime=1389151461.648933626
