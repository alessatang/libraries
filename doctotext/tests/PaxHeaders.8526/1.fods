30 mtime=1384978086.679497417
30 atime=1389151461.649933622
30 ctime=1389151461.649933622
