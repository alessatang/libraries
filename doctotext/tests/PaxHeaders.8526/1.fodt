30 mtime=1384978086.680497417
30 atime=1389151461.658933586
30 ctime=1389151461.658933586
