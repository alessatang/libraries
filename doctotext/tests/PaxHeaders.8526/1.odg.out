30 mtime=1384978086.681497418
30 atime=1389151462.011932196
30 ctime=1389151462.011932196
